api/OffscreenCanvas.json

```diff
@@ -0,0 +1,205 @@
+{
+  "api": {
+    "ImageData": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/OffscreenCanvas",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "60",
+            "notes": "This feature is behind a flag. In <code>chrome://flags</code> click <code>enable</code> under <code>Experimental canvas features</code>"
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "44",
+            "notes": "This feature is behind a feature preference setting. In about:config, set <code>gfx.offscreencanvas.enabled</code> to <code>true</code>."
```

Comment:
Please use a `flags` object instead of this note. (here and throughout the file)

Reply:
👍